resolwe_bio/processes/amplicon/bwa_trim.yml

```diff
@@ -9,11 +9,11 @@
     expression-engine: jinja
     executor:
       docker:
-        image: resolwebio/dnaseq:1.1.0
+        image: resolwebio/legacy:latest
     resources:
       memory: 32768
   data_name: "Alignment ({{ reads|sample_name|default('?') }})"
-  version: 0.1.0
+  version: 0.0.8
```

Comment:
The version should be higher then 0.1.0

Reply:
The process with version 0.1.0 was removed from the database, the current one is 0.0.7, right?

Reply:
Not on all deploys.